packages/typespec-test/test/openai_non_branded/generated/typespec-ts/src/models/models.ts

```diff
@@ -324,13 +324,13 @@ export interface Image {
   /** The URL of the generated image, if `response_format` is `url` (default). */
   url?: string;
   /** The base64-encoded JSON of the generated image, if `response_format` is `b64_json`. */
-  b64Json?: Uint8Array;
+  b64_Json?: Uint8Array;
 }
 
 export function imageDeserializer(item: any): Image {
   return {
     url: item["url"],
-    b64Json: !item["b64_json"]
+    b64_Json: !item["b64_json"]
```

Comment:
I prefer not keep the _ here, so we could produce more camelCase property, other wise we would see changes below:
`response_format` => `responseFormat` but `b64_json` to `b64_Json`.